File: array/1365_how_many_numbers_are_smaller_than_the_current_number.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    vector<int> smallerNumbersThanCurrent(vector<int>& nums) {
        vector<int> ans;
        for (auto ith : nums) {
            int ct = 0;
            for (auto jth : nums) {
                if (jth < ith) {
                    ct++;
                }
            }
            ans.push_back(ct);
        }
        return ans;
    }
};

int main() {
    
}

File: leetcode/py/1365_how_many_numbers_are_smaller_than_the_current_number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        ans = []
        for ith in nums:
            ct = 0
            for jth in nums:
                if jth < ith:
                    ct += 1
            ans.append(ct)
        return ans
